### PR TITLE
Make sure only public api methods are in generated docs

### PR DESCRIPTION
**Pull request by larrysalibra**
@moxiegirl This pull request exposes the method regarding transit key mentioned in #516 and also marks as private a number of blockchain methods that aren't usable at the moment by people making apps as we discussed.

If you have a moment, would love if you could take a look at this see if it makes sense.

### CODE REVIEW

File: src/auth/authApp.js [hunk 1]
```diff
@@ -27,25 +27,26 @@ const DEFAULT_PROFILE = {
...
-  return transitKey
+export function getTransitKey() : string {
+  const transitKey = localStorage.getItem(BLOCKSTACK_APP_PRIVATE_KEY_LABEL)
+  return ((transitKey: any): string)
 }
 
 /**
- * Fetches the hex value of the transit private key from local storage.
+ * Generates a ECDSA keypair to
+ * use as the ephemeral app transit private key
```
Comment (moxiegirl) [inline src/auth/authApp.js:41]: Maybe just "transit private key" which is the phrase you used for me. We can explain that a transit private key is ephemeral elsewhere.

**Review by moxiegirl — APPROVED** [review]
I left a change but it was a nit. If you want to ignore and merge, I can fix it later.

**Comment by x5engine** [comment]
what? a crow coding? @moxiegirl ?

**Comment by moxiegirl** [comment]
@meteorplus Yep.  We do lots of things humans don't expect.

**Pull request merged**